srcDoc/release_notes.rst

```diff
@@ -193,6 +193,15 @@ In previous releases, the `qos_string.h` file containing the strings from the
 `srcCpp03` folders. This process has been simplified, generating only once and
 into `srcCppCommon`.
 
+Add command-line argument to the `cmake` executable (PERF-194)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Starting in this release *RTI Perftest* enables the customer to add specific
+command-line arguments to the `cmake` executable (which is used when compiling
+against *RTI Connext DDS Micro*. This feature allows the control of the compiler
```

Comment:
A small typo: there is a missing closing parenthesis after "when compiling against *RTI Connext DDS Micro*"

Reply:
Done